arch/_common.sh

```diff
@@ -4,7 +4,7 @@
 
 # C Compiler Flags.
 CFLAGS_COMMON=('-pipe' '-Wno-error')
-CFLAGS_COMMON_OPTI=('-O2')
+CFLAGS_COMMON_OPTI=('-O2' '-fno-omit-frame-pointer' '-mno-omit-leaf-frame-pointer')
```

Comment:
Some CPU architectures may not have the concept of "frame pointers", enabling frame pointers on those platforms will cause the compiler to use an emulated version of "frame pointers" (while sacrificing an extra register for storing emulated frame pointer data).

Reply:
This is the case for almost *any* architecture. The concept of a frame pointer comes from the ABI, not from the architecture. In the case of x86-64, %rbp is one of 16 GP registers chosen to act as a frame pointer by ABI convention, and using `-fomit` frees it to be used for other purposes.